### PR TITLE
[17.0][IMP] sale_force_invoiced: Create specific group to isolate the access to the force invoiced action on sale order"

### PR DESCRIPTION
**Pull request by Anxo82**
Forward port: #3058

### CODE REVIEW

File: sale_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -4,12 +4,15 @@
...
     "summary": "Allows to force the invoice status of the sales order to Invoiced",
-    "version": "17.0.1.0.0",
+    "version": "17.0.1.1.0",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "category": "sale",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/sale-workflow",
-    "depends": ["sale"],
-    "data": ["view/sale_order.xml"],
+    "depends": ["sale", "sales_team"],
```
Comment (pedrobaeza) [inline sale_force_invoiced/__manifest__.py]: No need to include sales_team, as it's already a dependency of sale

**Review by pedrobaeza — APPROVED** [review]

File: sale_force_invoiced/__manifest__.py (hunk 1 above)
Comment (Anxo82) [reply sale_force_invoiced/__manifest__.py]: Done :+1:

**Comment by dalonsod** [comment]
@pmonje-duonex could you review? Thanks!

**Review by Jaimermaccione — APPROVED** [review]
LGTM! Functional review on runboat

**Review by pmonje-duonex — APPROVED** [review]
LGTM! Functional review

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**